src/trainer/evaluator.py

```diff
@@ -120,19 +131,19 @@ def _run_eval_and_plot(self, agent: Agent, state: TrainerState, final=False):
         self._plot_results()
 
     def _create_env(self, atari=False):
-        env_name = self.user_args["env_name"]
-        if atari:
-            game = env_name.split(":")[1]
-            _env = AtariEnv(self.device, game, self.user_args["seed"])
-            return AtariVisualiser(
-                _env,
-                self.output_dir,
-                filename=f"tmp",
-                seed=self.user_args["seed"],
-            )
-
-        _env = gym.make(self.user_args["env_name"], render_mode="rgb_array")
-        return Visualiser(_env, self.output_dir, filename=f"tmp", seed=self.user_args["seed"])
+        env_name = self.user_args["level"]
+        # if atari:
+        #     game = env_name.split(":")[1]
+        #     _env = AtariEnv(self.device, game, self.user_args["seed"])
+        #     return AtariVisualiser(
+        #         _env,
+        #         self.output_dir,
+        #         filename=f"tmp",
+        #         seed=self.user_args["seed"],
+        #     )
+
+        _env = gym.make("BabyAI-GoToRedBallGrey-v0", render_mode="rgb_array")
```

Comment:
hard coded this config and seed 0 (one of the validation seeds for this config) for testing purposes - IRL this is where we'd be dynamically accessing the different available configs and validation/test seeds